doc/dev-jumpstart.md

```diff
@@ -28,20 +28,20 @@ or integrating with the pipeline.
 
 Other directories include:
 
-* `apline`: contains Docker image used by some tests
+* `alpine`: contains the Docker image used by some tests
 * `art`: contains graphical resources used in documentation
 * `asm`: contains assembly code
-* `crt`: contains Mystikos's modification to musl library
-* `doc`: contains documentation
-* `docker`: contains file to build Mystikos on Docker
+* `crt`: contains Mystikos's addition to musl library
+* `doc`: contains documentations
+* `docker`: contains files to build Mystikos in a container
 * `ext2`: contains EXT2 file system implementation
-* `gcov`: contains gcov code coverage tool
-* `host`: contains files used by Mystikos host
-* `hostfs`: a file system not in use
-* `include`: header files
-* `json`: a json parser implementation
-* `oe`: contains extension to OpenEnclave
-* `prereqs`: prerequisite
+* `gcov`: contains gcov-based code coverage utilities
+* `host`: contains files used by the host launcher of Mystikos
+* `hostfs`: a file system primarily used for testing/debugging
+* `include`: contains Mystikos header files
+* `json`: a json parser implementation, primarily used to parse configuration file
+* `oe`: contains extension to OpenEnclave, exposing OE APIs through syscalls
+* `prereqs`: contains prerequisite such as OpenEnclave driver and development dependencies
```

Comment:
`OpenEnclave driver and development dependencies` -> `Intel SGX driver and other required application packages`